src/OpenCL.cpp

```diff
@@ -750,11 +750,11 @@ void OpenCL_Network::convolve1(int channels, int outputs,
                               cl::Buffer& bufferOutput,
                               cl::Buffer& bufferMerge,
                               weight_slice_t weights) {
-    // fixed for 19x19
-    constexpr int width = 19;
-    constexpr int height = 19;
+    // fixed for BOARD_SIZExBOARD_SIZE
```

Comment:
It's no longer fixed now is it? :-)

Reply:
Ah yeah, I'll modify the comment. By the wawy, should I keep these variables in those functions or should I remove them and directly use BOARD_SIZE/BOARD_SQUARES in the code?

Reply:
Keep the variables for now.